GitHubVS.sln

```diff
@@ -2,6 +2,8 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.26228.4
```

Comment:
Let's avoid doing these changes, they're just going to pop up again for the next person.